src/frontend/download-chrome.ts

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
+import * as os from "os";
+import { promisify } from "util";
+import * as fs from "fs";
+import got from "got";
+import stream from "node:stream";
+import extract from "extract-zip";
+import { unreachable } from "./src/utils/utils";
+
+const pipeline = promisify(stream.pipeline);
+
+const stat = promisify(fs.stat);
+const unlink = promisify(fs.unlink);
+
+const installPath = `${os.homedir()}/.local-chromium`;
+
+type Config = { platform: "linux" | "darwin"; revision: string };
+
+/* This downloads Chromium from the Chromium build archives
+ * For more info on finding a particular version of chromium, see https://www.chromium.org/getting-involved/download-chromium/
```

Comment:
maybe link also the tool of Cypress which ease the process of finding a version: https://chromium.cypress.io/

Reply:
oh this is NICE!

Reply:
Added